### ISSUE TITLE
Clarify one comment about not using think time

### ISSUE THREAD

**Issue by MattDodsonEnglish** [grafana/k6-learn#9]
The "Think time" doc says that users _might not_ use want think time when:

> Your load generator can run your test script without crossing the 80% CPU utilization mark.

Why is this the case? It's not so obvious to me how this pertains to the decision to use `sleep`.

**Comment by nicolevanderhoeven** [comment]
Good point. I tried to explain it in the previous section, but it could use some work.

How about [this](https://github.com/grafana/k6-learn/commit/c204b1ad533522e158b537d419c0de82742d9e6f)?
